lib/config-migrator.js

```diff
@@ -0,0 +1,59 @@
+/** @babel */
+
+import _ from 'lodash'
+import { heredoc } from './werkzeug'
+
+export default async function checkConfigAndMigrate () {
```

Comment:
But of course, if making `activate` async works without side-effects, then we might as well just do that I guess. I really think we need to test that assumption though. AFAIK package activation was never designed to be async.